locales/en/allocations.json

```diff
@@ -24,5 +24,40 @@
   },
   "allocation_criteria": {
     "page_title": "Allocation criteria"
+  },
+  "view": {
+    "page_title": "View allocation",
+    "heading": "{{moves_count}} {{person_or_people}} from {{from_location}} to {{to_location}}, {{date}}",
+    "criteria_for_allocation": "Criteria for allocation",
+    "created_at": "Created {{datetime}}",
+    "allocation_tab": "Allocation",
+    "allocation_history_tab": "Allocation history",
+    "fields": {
```

Comment:
Could we re-use the labels from the fields translation file?

Reply:
The wording is different -- I believe, intentionally.

Reply:
I would check with @Andy-Millar but I feel like it might be better to have consistency in labelling between capture and replay if we can.

Reply:
@Andy-Millar we have two sets of wordings for the sentence length. In the radio buttons, we have
```
      "long": "Over 16 months only",
      "short": "Under 16 months only",
      "any": "Any sentence"
```
Whilst in the detail view of the allocation, we have:
```
        "any": "Any length of sentence",
        "short": "16 months or less to serve",
        "long": "16 months or longer to serve"
```
Would it be possible to stick with one set for both?

Reply:
@barbaracassani 

Worth noting this copy was updated a while ago in the clickable prototype - I'm guessing you built this from a screenshot which is out of date. Going forward we have been working on links to prototype rather than screenshot, so hopefully this will happen less and less going forward. 

Here's the page in prototype: https://invis.io/YET7ANHWV9K#/412212287_IPTC-MVP-03 

Copy in prototype for PMU creating a move is:

- "any": "Any time to serve"
- "short": "16 months or less",
- "long": "Over 16 months",

For the allocation page we should have the same - though the invision prototype does have the words "to serve" at the end, we can remove them - just to clarify it should be:

- "any": "Any time to serve"
- "short": "16 months or less",
- "long": "Over 16 months"

Reply:
Ok, thank you for the clarification @Andy-Millar -- shall amend.